Tasks/NodeToolV0/nodetool.ts

```diff
@@ -5,12 +5,13 @@ import * as os from 'os';
 import * as path from 'path';
 
 let osPlat: string = os.platform();
-let osArch: string = os.arch();
+let architecture: string;
```

Comment:
please leave the var name osArch since it represents os.arch()

Reply:
With this change it doesn't anymore. EX: the value might be `x86` even if the os.arch is actually `x64`

Reply:
leave the variable name osArch

Reply:
OK, it's your project